src/styles/components/_NotFoundButton.scss

```diff
@@ -0,0 +1,48 @@
+.notfound-404 {
+    position: relative;
+    height: 200px;
+    h2 {
+      font-family: 'Montserrat', sans-serif;
+      font-size: 22px;
+      font-weight: 700;
+      text-transform: uppercase;
+      color: #ffffff;
+      margin-top: 20px;
+      margin-bottom: 15px;
+    }
+  }
+
+  .home-btn {
+    font-family: 'Montserrat', sans-serif;
+    display: inline-block;
+    font-weight: 700;
+    text-decoration: none;
+    background-color: transparent;
+    border: 2px solid transparent;
+    text-transform: uppercase;
+    padding: 13px 25px;
+    font-size: 18px;
+    border-radius: 40px;
+    margin: 7px;
+    -webkit-transition: 0.2s all;
+    transition: 0.2s all;
+  }
+  .home-btn {
+    color: #1ABECB;
```

Comment:
Hi @TheDrizzyWay, this does not seem to be one of the colors in the color palette we used in our designs. I think we should stick to our color palette for uniformity

Reply:
Thanks for catching this. I will get the links to the palettes from @davidokonji and fix it.